src/main/java/edu/hawaii/its/api/controller/GroupingsRestController.java

```diff
@@ -122,6 +122,9 @@ public ResponseEntity<String> getGrouping(@RequestBody(required = true) List<Str
             @RequestParam(required = true) String sortBy,
             @RequestParam(required = true) Boolean isAscending) {
         logger.info("Entered REST getGrouping...");
+        String groupPathsStr = groupPaths != null ? groupPaths.toString() : "null";
+        logger.info(String.format("Received Parameters - groupPaths: %s, page: %d, size: %d, sortBy: %s, isAscending: %b",
+                groupPathsStr, page, size, sortBy, isAscending));
         String currentUid = policy.sanitize(userContextService.getCurrentUid());
```

Comment:
Could you include the currentUid in the logger message, preferably at the beginning? 

Also it would be beneficial to log the parameters of the rest of the endpoints.

Thanks!